src/auxkernels/ProcRateForRateCoeffThreeBody_Crane.C

```diff
@@ -0,0 +1,48 @@
+//* This file is part of Zapdos, an open-source
+//* application for the simulation of plasmas
+//* https://github.com/shannon-lab/zapdos
+//*
+//* Zapdos is powered by the MOOSE Framework
+//* https://www.mooseframework.org
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ProcRateForRateCoeffThreeBody_Crane.h"
+
+registerMooseObject("ZapdosApp", ProcRateForRateCoeffThreeBody_Crane);
```

Comment:
This needs to be changed to "CraneApp" instead of "ZapdosApp". (This change should be applied to every file where "ZapdosApp" is specified, not just this line.)